### PR TITLE
Feature#19 Designs Login and Sign Up

### LINKED ISSUE #19: Add design to sign up
# REQUIREMENTS

- follow the mockup design 

# Acceptance Criteria
List all required acceptance criteria
- [ ] still working correctly

# Design
home-signUp

### PR DESCRIPTION
**Pull request by AngelBFdev**
# Description
- Added Sign Up
- Updated Login Design
  
  ## Screenshots 
Mockup Login
image
Frontend Login
image

Mockup Sign up
image
Frontend Sign up
image

### CODE REVIEW

File: app/assets/stylesheets/application.scss [hunk 1]
```diff
@@ -1,10 +1,21 @@
...
   font-family: $poppins;
   margin: 0;
   width: 100%;
 }
+
+html {
+  font-size: 62.5%;
+  box-sizing: border-box;
+  scroll-padding-top: 0rem;
+}
```
Comment (JuanequeX) [inline app/assets/stylesheets/application.scss]: What is the reason to have two html label styles?

File: app/assets/stylesheets/components/_login.scss [hunk 2]
```diff
@@ -1,49 +1,66 @@
...
 
-      .button__sigIn{
-        background-color: $feed_card;
-        color: $color_white;
-        height: 3.5rem;
-        margin-top: 2rem;
-        width: 17rem;
+      .link-container{
+        margin-top: -0.5rem;
+        .button__sigIn{
```
Comment (JuanequeX) [inline app/assets/stylesheets/components/_login.scss]: this is a bem nomenclature, please use it or change the name of this className

**Review by JuanequeX — CHANGES_REQUESTED** [review]

File: app/assets/stylesheets/application.scss (hunk 1 above)
Comment (Villaedi) [reply app/assets/stylesheets/application.scss]: done

File: app/assets/stylesheets/components/_login.scss (hunk 2 above)
Comment (Villaedi) [reply app/assets/stylesheets/components/_login.scss]: done

**Review by YilKanda — APPROVED** [review]
LGTM!

**Review by JuanequeX — APPROVED** [review]
LGTM

**Review by IrenehMDev — APPROVED** [review]
LGTM

**Pull request merged**